provision-osx/defaults/main.yml

```diff
@@ -4,6 +4,11 @@
 
 remote_tmp_dir: /tmp
 
+git_install_url: https://netix.dl.sourceforge.net/project/git-osx-installer/git-2.13.1-intel-universal-mavericks.dmg
```

Comment:
I used https://sourceforge.net/projects/git-osx-installer/files/git-2.13.1-intel-universal-mavericks.dmg/download?use_mirror=autoselect and it worked.

of course, you have to rename the output file to something.dmg